webpages/js/metaanalyses.js

```diff
@@ -2345,6 +2441,161 @@
   }
 
 
+/* BANNER HERE FOR 'graphs'
```

Comment:
it seems you started this work on a topic branch rather than master.

The graph changes here have been implemented to master already, so they shouldn't be appearing here.

The (probably best.. if you know another go for it) remedy for this is to get this branch in order, then cherrypick whatever commits you want to keep onto a new branch (with the new branch being based off an up-to-date master, i.e. `git fetch --all; git checkout master; git pull upstream master`).

If you get stuck, or have questions here ask for help.